frontend/js/components/environments/Home.js

```diff
@@ -24,12 +31,14 @@ class Home extends Component {
 }
 
 Home.propTypes = {
-    user: PropTypes.object
+    user: PropTypes.object,
```

Comment:
It would be coming from the global store through `connect` at the bottom of `Home.js` (or any other environment).  What `user` is doing isn't well defined yet, so I'll remove it for now.